dashboards/Analyze.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Should I still have a ```dashboards/``` directory or should I just copy the larger json file into the kibana repo by itself